Move Security section to the end. Fix HTML errors.

zolkis opened the pull request:
To editors: please take attention to close tags. Browsers do complete them, but tools will complain.
Also, no flow content inside `<p>`.
Will continue with the Security section in a subsequent PR, this is to have a corrected baseline.
Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/zolkis/web-nfc/pull/436.html" title="Last updated on Nov 6, 2019, 8:15 AM UTC (59075ad)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/436/bc06589...zolkis:59075ad.html" title="Last updated on Nov 6, 2019, 8:15 AM UTC (59075ad)">Diff</a>

Commit:
Move Security section to the end. Fix HTML errors.

Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>

kenchris commented on index.html:
```diff
@@ -4253,9 +4100,175 @@ <h3>Parsing content</h3>
+
+  <section> <h3>Chain of trust</h3>
+  <p>
+    Web pages using Web NFC are not trusted.
```
considered trusted

zolkis replied:
Content of Security section will be updated in a future PR.

kenchris commented on index.html:
```diff
@@ -4253,9 +4100,175 @@ <h3>Parsing content</h3>
+    Web pages using Web NFC are not trusted.
+    This means that the user needs to be aware of exactly what a web page is
+    intending to do with NFC at any given moment. Implementations need to
+    make sure that when the user authorizes a method of this API, then only that
```
'authorizes a method of this API' sounds cryptic

zolkis replied:
same as above

kenchris commented on index.html:
```diff
@@ -4253,9 +4100,175 @@ <h3>Parsing content</h3>
+    </section>
+    <section> <h4>Restrict automatic handling</h4>
+      <p>
+        The payload data on <a>NFC content</a> is untrusted, and must not be used
```
But people can use signature records to make it trusted

zolkis replied:
> a malicious third party could delete the signature record from the NDEF message or
attach a new signature record to prevent the user from noticing any malicious change of content.

zolkis replied:
But with the right policies in place it should be indeed fine (trusted). 
Anyway, content changes come in a next PR.

beaufortfrancois commented:
Can you split this CL in two CLs so that it's easier to review?

zolkis commented:
There is no content change yet, just moved the Security section to the end, added missing closing tags and changed `<p>` tags that contained flow content. 

Purely syntactic changes, in that sense, this is the split, before I make content changes.

beaufortfrancois commented:
I was confused as @kenchris was suggesting changes.
Did you run a tool to make sure all html tags were closed?

zolkis commented:
> Did you run a tool to make sure all html tags were closed?

Good idea. I've run tidy and fixed a few more errors.

beaufortfrancois reviewed (approved):
LGTM
